src/main/scala/org/scalameter/Persistor.scala

```diff
@@ -6,15 +6,15 @@ package org.scalameter
 
 
 trait Persistor {
-  def load(context: Context): History
-  def save(context: Context, h: History): Unit
+  def load[T: Manifest](context: Context): History[T]
+  def save[T](context: Context, h: History[T]): Unit
```

Comment:
It's possible that the `include` statement got broken at some point.

The bench that has the `include` statement should execute all the snippets from the included benchmarks. If there are multiple `include` statements, then all the tests from the sub-benchmarks should be included.

With the current refactoring that introduces bench result types, and the recent mailing list discussion, we would need to think about how we want to execute those nested tests and report their results.
I'm fine if we fix all that later, in a separate PR.